robot.py

```diff
@@ -300,6 +304,9 @@ def testPeriodic(self):
                     absolute_rotation=True,
                 )
 
+        if self.gamepad.getTriggerAxis(self.gamepad.Hand.kLeft) > 0.5:
+            self.hatch_wedge_piston.set(2)
```

Comment:
You probably want to use `wpilib.DoubleSolenoid.Value.kReverse` so that you can tell whether it makes the solenoid go backwards or forwards.